### PR TITLE
add test for api/blocklistener.go and api/listener.go

### PR DESCRIPTION
**Pull request by newyear6565**

### CODE REVIEW

File: api/blocklistener_test.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		{
+			BlockHeight: 2,
+		},
+	}
+	builder := block.NewTestingBuilder()
+	builder.SetHeight(1)
+	builder.SetVersion(111)
+	builder.SetTimeStamp(time.Now())
+	builder.SetReceipts(receipts)
+	testBlock, _ := builder.SignAndBuild(identityset.PrivateKey(0))
```
Comment (koseoyoung) [inline api/blocklistener_test.go:49]: catch error here?

File: api/blocklistener_test.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+	server.EXPECT().Send(gomock.Any()).Return(nil).Times(1)
+	err := responder.Respond(&testBlock)
+	require.NoError(t, err)
```
Comment (koseoyoung) [inline api/blocklistener_test.go:53]: ```require.NoError(t, responder.Respond(&testBlock)) ```

Comment (koseoyoung) [reply api/blocklistener_test.go:53]: same comments on below as well

Comment (newyear6565) [reply api/blocklistener_test.go]: > `require.NoError(t, responder.Respond(&testBlock)) `

ok ,it has been done

File: api/blocklistener_test.go (hunk 1 above)
Comment (newyear6565) [reply api/blocklistener_test.go]: > catch error here?

ok ,it has been done

**Pull request merged**